src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -1539,17 +1515,16 @@ void PSParallelCompact::forward_to_new_addr() {
 #ifdef ASSERT
 void PSParallelCompact::verify_forward() {
```

Comment:
Could this method become `const` now?

Reply:
It's a `static` method; `const` is only for non-static ones.